modules/ui/src/main/scala/lucuma/ui/forms/EnumViewSelectBase.scala

```diff
@@ -210,7 +210,7 @@ object EnumViewSelectBase {
           p.tpe,
           p.trigger,
           p.upward,
-          value = p.getter,
+          value = p.getter.getOrElse(null),
```

Comment:
I'm a bit surprised this is necessary. Passing `undefined` here would make the facade omit the value, which I would expect would unset. That isn't the case? Does the `Select` get stuck with the last value?

Reply:
It is surprising to me too, I also expected  undefined to work but it keeps the previous value